client.go

```diff
@@ -491,7 +481,7 @@ func (c *Client) sendHub() {
 				// And get rid of the timeout workaround.
 				c.LocalKite.Log.Error("error sending %q: %s", msg, err)
 
-				if err == sockjsclient.ErrSessionClosed {
+				if err == sockjsclient.ErrSessionClosed || strings.Contains(err.Error(), errClosing) {
```

Comment:
hmm, `errClosing` should be an error from our end (in this case Client) not from remote side. If server to return or bubble `errClosing` err of it to their clients, that could be the case but as i checked the Kite Server when we get that error, we simply exit from accepting new connections. In the case where server is in closing/closes/closed states, most probably we could get a syscal error like "Connection reset by peer"